Themis/Models/Submission.swift

```diff
@@ -38,7 +38,8 @@ struct SubmissionParticipation: Codable {
     let id: Int
     let repositoryUrl: String
     let userIndependentRepositoryUrl: String
-    let student: Student
+    let exercise: ExerciseOfSubmission
+    // let student: Student
```

Comment:
Why is this commented?

Reply:
Doesn't appear in the response of a random submission. Will figure this out when I continue the implementation.